src/curve.rs

```diff
@@ -231,6 +231,9 @@ pub trait Identity {
     /// Returns the identity element of the curve.
     /// Can be used as a constructor.
     fn identity() -> Self;
+
```

Comment:
Is there a good reason not to just have `is_identity()` part of `impl ExtendedPoint`?